api.go

```diff
@@ -8,26 +8,26 @@ import (
 
 	"github.com/maniksurtani/quotaservice/config"
 	"github.com/maniksurtani/quotaservice/logging"
-
-	pbconfig "github.com/maniksurtani/quotaservice/protos/config"
+	pb "github.com/maniksurtani/quotaservice/protos/config"
 )
 
 // The Server interface is what you get when you create a new quotaservice.
 type Server interface {
 	Start() (bool, error)
 	Stop() (bool, error)
 	SetLogger(logger logging.Logger)
-	ServeAdminConsole(*http.ServeMux, string, config.ConfigPersister, bool)
+	CreateBucketContainer(*pb.ServiceConfig)
```

Comment:
Why is this method public on `Server`? Shouldn't it either be part of constructing a `Server` (i.e., in `New()`), or when starting a `Server` (i.e., in `Start()`)?